### PR TITLE
fix(*): cleanup unused code and dependencies

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: src/packages/client/scripts/postinstall.js [hunk 1]
```diff
@@ -322,8 +325,6 @@ function parsePackageManagerName(userAgent) {
   return packageManager
 }
 
-// prettier-ignore
-const UNABLE_TO_FIND_POSTINSTALL_TRIGGER__EMPTY_STRING = 'UNABLE_TO_FIND_POSTINSTALL_TRIGGER__EMPTY_STRING'
```
Comment (williamluke4) [inline src/packages/client/scripts/postinstall.js:326]: Is this not used?

**Review by williamluke4 — APPROVED** [review]
Nice work Joël! :tada:

File: src/packages/client/scripts/postinstall.js (hunk 1 above)
Comment (Jolg42) [reply src/packages/client/scripts/postinstall.js:326]: I could not find it used anywhere, I guess it was used before and not anymore.

**Comment by Jolg42** [comment]
I think we can forget about the flaky GitHub Actions test, I already did a rerun and same thing... 🤷‍♂️

**Comment by timsuchanek** [comment]
Good work! I did not forget about the flaky GH actions test, I'm actually working on it today, so soon this will not come anymore

**Review by timsuchanek — APPROVED** [review]

File: src/packages/client/scripts/postinstall.js (hunk 1 above)
Comment (janpio) [reply src/packages/client/scripts/postinstall.js:326]: Ping @jasonkuhrt - leftover?

Comment (jasonkuhrt) [reply src/packages/client/scripts/postinstall.js:326]: Hey, probably not a big deal

**Pull request merged**